lib/cluster/deployment.rb

```diff
@@ -63,7 +63,7 @@ def self.execute_chef_recipes_on_layers(recipes: [], layers: [], custom_json: ''
     end
 
     def self.update_dependencies
-      run_command_on_layers(command: 'dependencies')
+      run_command_on_layers(command: 'update_dependencies')
```

Comment:
We're using Chef 11.10 now and the docs say "Starting in Chef 12, this command [Update Dependencies] is not available"  and suggests creating a custom recipe for non-boot time updates. Is there a reason we should look out for on why it's being discontinued? http://docs.aws.amazon.com/opsworks/latest/userguide/workingstacks-commands.html